packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -186,6 +239,11 @@ contract AnchorStateRegistry is Initializable, ISemver {
             return false;
         }
 
+        // Must not be paused, temporarily causes game to be considered improper.
+        if (paused()) {
```

Comment:
return `!paused()` has less branches if you add a comment explaining why

Reply:
I kinda like the symmetry of the function with the branches, will leave as-is